### PR TITLE
Definition lists for accessibility

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: N/A

Description of Changes:
* Changing pseudo-definition lists into actual definition lists in the configuration snippets

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: _includes/configuration/a11ychecker_issue_url_callback.md [hunk 1]
```diff
@@ -2,9 +2,11 @@
 
 The `a11ychecker_issue_url_callback` option is used to change the target URL for the "Click for more info" button (![help icon - a question mark inside a circle]({{site.baseurl}}/images/icons/help.svg)) in the Accessibility Checker dialog. By default, the "more info" links will point to `{{site.url}}/docs/plugins/a11ychecker/#<ruleId>`, such as `{{site.url}}/docs/plugins/a11ychecker/#D1`. This option can be used to set the target URL to a page or pages outside {{site.url}}/docs/.
 
-**Type:** `Function`
+Type
+: `Function`
 
-**Default Value:** `undefined`
+Default Value
+: `undefined`
```
Comment (lnewson) [inline _includes/configuration/a11ychecker_issue_url_callback.md:9]: Not something new, but we really should delete this default, as it being `undefined` means it has no default not that that's the value.

File: _includes/integrations/react-tech-ref.md [hunk 2]
```diff
@@ -513,9 +536,11 @@ Used to set the toolbar for the editor. Using `<Editor toolbar='bold' />` is the
 
 For information setting the toolbar for {{site.productname}}, see: [User interface options - toolbar]({{site.baseurl}}/configure/editor-appearance/#toolbar).
 
-**Possible values:**  See [Toolbar Buttons Available for {{site.productname}}]({{site.baseurl}}/advanced/available-toolbar-buttons/).
+Possible values
+: See [Toolbar Buttons Available for {{site.productname}}]({{site.baseurl}}/advanced/available-toolbar-buttons/).
 
-**Type:** String
+Type
+: String
```
Comment (lnewson) [inline _includes/integrations/react-tech-ref.md:543]:
```suggestion
: `String`
```

File: _includes/integrations/react-tech-ref.md [hunk 3]
```diff
@@ -530,7 +555,8 @@ For information setting the toolbar for {{site.productname}}, see: [User interfa
 
 Use the `tinymceScriptSrc` prop to specify an external version of {{site.productname}} to lazy load.
 
-**Type:** String
+Type
+: String
```
Comment (lnewson) [inline _includes/integrations/react-tech-ref.md:559]:
```suggestion
: `String`
```

File: _includes/integrations/react-tech-ref.md [hunk 4]
```diff
@@ -555,7 +581,8 @@ the `value` prop and using the `onEditorChange` event to update the `value`.
 
 For detailed information on using the `value` prop, see: [Using the {{site.productname}} React component as a controlled component](#usingthetinymcereactcomponentasacontrolledcomponent).
 
-**Type:** String
+Type
+: String
```
Comment (lnewson) [inline _includes/integrations/react-tech-ref.md:585]:
```suggestion
: `String`
```

File: _includes/integrations/vue-tech-ref.md [hunk 5]
```diff
@@ -104,9 +104,11 @@ None of the configuration properties are **required** for `tinymce-vue` to work.
 
 {% include misc/get-an-api-key.md %}
 
-**Default value:** `no-api-key`
+Default value
+: `no-api-key`
 
-**Type:** String
+Type
+: String
```
Comment (lnewson) [inline _includes/integrations/vue-tech-ref.md:111]:
```suggestion
: `String`
```

File: _includes/integrations/vue-tech-ref.md [hunk 6]
```diff
@@ -118,9 +120,11 @@ None of the configuration properties are **required** for `tinymce-vue` to work.
 
 #### `cloud-channel`
 
-**Default value:** `{{site.productmajorversion}}`
+Default value
+: `{{site.productmajorversion}}`
```
Comment (lnewson) [inline _includes/integrations/vue-tech-ref.md:124]:
```suggestion
: `'{{site.productmajorversion}'}`
```

File: _includes/integrations/vue-tech-ref.md [hunk 7]
```diff
@@ -118,9 +120,11 @@ None of the configuration properties are **required** for `tinymce-vue` to work.
...
 
-**Possible values:**  `{{site.productmajorversion}}-stable`, `{{site.productmajorversion}}-testing`, `{{site.productmajorversion}}-dev`
+Possible values
+: `{{site.productmajorversion}}-stable`, `{{site.productmajorversion}}-testing`, `{{site.productmajorversion}}-dev`
```
Comment (lnewson) [inline _includes/integrations/vue-tech-ref.md:127]:
```suggestion
: `'{{site.productmajorversion}}-stable'`, `'{{site.productmajorversion}}-testing'`, `'{{site.productmajorversion}}-dev'`
```

File: _includes/integrations/vue-tech-ref.md [hunk 8]
```diff
@@ -157,9 +163,11 @@ The `disabled` property can dynamically switch the editor between a "disabled" (
 #### `id`
 An id for the editor. Used for retrieving the editor instance using the `tinymce.get('ID')` method. Defaults to an automatically generated [UUID](https://tools.ietf.org/html/rfc4122).
 
-**Default value:** Automatically generated [UUID](https://tools.ietf.org/html/rfc4122).
+Default value
+: Automatically generated [UUID](https://tools.ietf.org/html/rfc4122).
 
-**Type:** String
+Type
+: String
```
Comment (lnewson) [inline _includes/integrations/vue-tech-ref.md:170]:
```suggestion
: `String`
```

File: _includes/integrations/vue-tech-ref.md [hunk 9]
```diff
@@ -174,9 +182,11 @@ Object sent to the `tinymce.init` method used to initialize the editor.
 
 For information on the {{site.productname}} selector (`tinymce.init`), see: [Basic setup]({{site.baseurl}}/general-configuration-guide/basic-setup/).
 
-**Default value:** `"{% raw %}{{% endraw %} {% raw %}}{% endraw %}"`
+Default value
+: `"{% raw %}{{% endraw %} {% raw %}}{% endraw %}"`
 
-**Type:** Object
+Type
+: Object
```
Comment (lnewson) [inline _includes/integrations/vue-tech-ref.md:189]:
```suggestion
: `Object`
```

File: _includes/integrations/vue-tech-ref.md [hunk 10]
```diff
@@ -194,9 +204,11 @@ For information on the {{site.productname}} selector (`tinymce.init`), see: [Bas
 #### `initial-value`
 Initial content of the editor when the editor is initialized.
 
-**Default value:** `" "`
+Default value
+: `" "`
 
-**Type:** String
+Type
+: String
```
Comment (lnewson) [inline _includes/integrations/vue-tech-ref.md:211]:
```suggestion
: `String`
```

File: _includes/integrations/vue-tech-ref.md [hunk 11]
```diff
@@ -229,9 +243,11 @@ Sets the trigger events for [v-model events](#forminputbindingsv-model).
 
 For a list of available {{site.productname}} events, see: [Available Events - Editor events]({{site.baseurl}}/advanced/events/#editorcoreevents).
 
-**Default value:** `"change keyup undo redo"`.
+Default value
+: `"change keyup undo redo"`.
 
-**Type:** String
+Type
+: String
```
Comment (lnewson) [inline _includes/integrations/vue-tech-ref.md:250]:
```suggestion
: `String`
```

File: _includes/integrations/vue-tech-ref.md [hunk 12]
```diff
@@ -229,9 +243,11 @@ Sets the trigger events for [v-model events](#forminputbindingsv-model).
 
 For a list of available {{site.productname}} events, see: [Available Events - Editor events]({{site.baseurl}}/advanced/events/#editorcoreevents).
 
-**Default value:** `"change keyup undo redo"`.
+Default value
+: `"change keyup undo redo"`.
```
Comment (lnewson) [inline _includes/integrations/vue-tech-ref.md:247]:
```suggestion
: `'change keyup undo redo'`.
```

File: _includes/integrations/vue-tech-ref.md [hunk 13]
```diff
@@ -245,11 +261,14 @@ For a list of available {{site.productname}} events, see: [Available Events - Ed
 
 Used to specify the format of the content emitted via the `input` event. This affects the format of the content used in conjunction with data binding.
 
-**Type:** String
+Type
+: String
```
Comment (lnewson) [inline _includes/integrations/vue-tech-ref.md:265]:
```suggestion
: `String`
```

File: _includes/integrations/vue-tech-ref.md [hunk 14]
```diff
@@ -245,11 +261,14 @@ For a list of available {{site.productname}} events, see: [Available Events - Ed
...
 
-**Default value:** `html`
+Default value
+: `html`
```
Comment (lnewson) [inline _includes/integrations/vue-tech-ref.md:268]:
```suggestion
: `'html'`
```

File: _includes/integrations/vue-tech-ref.md [hunk 15]
```diff
@@ -245,11 +261,14 @@ For a list of available {{site.productname}} events, see: [Available Events - Ed
...
 
-**Possible values:** `html`, `text`
+Possible values
+: `html`, `text`
```
Comment (lnewson) [inline _includes/integrations/vue-tech-ref.md:271]:
```suggestion
: `'html'`, `'text'`
```

File: _includes/integrations/vue-tech-ref.md [hunk 16]
```diff
@@ -264,7 +283,8 @@ Used to include plugins for the editor. Using `<editor plugins="lists code" />`
 
 For information on adding plugins to {{site.productname}}, see: [Add plugins to {{site.productname}}]({{site.baseurl}}/plugins/).
 
-**Type:** String or Array
+Type
+: String or Array
```
Comment (lnewson) [inline _includes/integrations/vue-tech-ref.md:287]:
```suggestion
: `String` or `Array`
```

File: _includes/integrations/vue-tech-ref.md [hunk 17]
```diff
@@ -277,9 +297,11 @@ For information on adding plugins to {{site.productname}}, see: [Add plugins to
 #### `tag-name`
 Only valid when [`<editor :inline=true />`](#inline). Used to define the HTML element for the editor in inline mode.
 
-**Default value:** `div`
+Default value
+: `div`
 
-**Type:** String
+Type
+: String
```
Comment (lnewson) [inline _includes/integrations/vue-tech-ref.md:304]:
```suggestion
: `String`
```

File: _includes/integrations/vue-tech-ref.md [hunk 18]
```diff
@@ -277,9 +297,11 @@ For information on adding plugins to {{site.productname}}, see: [Add plugins to
 #### `tag-name`
 Only valid when [`<editor :inline=true />`](#inline). Used to define the HTML element for the editor in inline mode.
 
-**Default value:** `div`
+Default value
+: `div`
```
Comment (lnewson) [inline _includes/integrations/vue-tech-ref.md:301]:
```suggestion
: `'div'`
```

File: _includes/integrations/vue-tech-ref.md [hunk 19]
```diff
@@ -295,9 +317,11 @@ Used to set the toolbar for the editor. Using `<editor toolbar="bold italic" />`
 
 For information setting the toolbar for {{site.productname}}, see: [User interface options - toolbar]({{site.baseurl}}/configure/editor-appearance/#toolbar).
 
-**Possible values:**  See [Toolbar Buttons Available for {{site.productname}}]({{site.baseurl}}/advanced/available-toolbar-buttons/).
+Possible values
+: See [Toolbar Buttons Available for {{site.productname}}]({{site.baseurl}}/advanced/available-toolbar-buttons/).
 
-**Type:** String
+Type
+: String
```
Comment (lnewson) [inline _includes/integrations/vue-tech-ref.md:324]:
```suggestion
: `String`
```

File: _includes/integrations/vue-tech-ref.md [hunk 20]
```diff
@@ -311,7 +335,8 @@ For information setting the toolbar for {{site.productname}}, see: [User interfa
 #### `tinymce-script-src`
 Use the `tinymce-script-src` prop to specify an external version of TinyMCE to lazy load.
 
-**Type:** String
+Type
+: String
```
Comment (lnewson) [inline _includes/integrations/vue-tech-ref.md:339]:
```suggestion
: `String`
```

**Review by lnewson — APPROVED** [review]
All the suggestions are for pre-existing issues but as this is about accessibility then we should be making sure these are marked up correctly as well.

**Review by EkimChau — APPROVED** [review]

File: _includes/integrations/react-tech-ref.md [hunk 21]
```diff
@@ -187,9 +189,11 @@ Required for deployments using the {{site.cloudname}} to provide the {{site.prod
 
 Changes the {{site.productname}} build used for the editor to either a specific version or a channel indicating a stability level.
 
-**Default value:** `{{site.productmajorversion}}-stable`
+Default value
+: `'{{site.productmajorversion}}-stable'`
 
-**Possible values:**  `{{site.productmajorversion}}-stable`, `{{site.productmajorversion}}-testing`, `{{site.productmajorversion}}-dev`, `{{site.productminorversion}}`
+Possible values
+: `'{{site.productmajorversion}}-stable'`, `'{{site.productmajorversion}}-testing'`, `'{{site.productmajorversion}}-dev`, `{{site.productminorversion}}'`
```
Comment (lnewson) [inline _includes/integrations/react-tech-ref.md:196]:
```suggestion
: `'{{site.productmajorversion}}-stable'`, `'{{site.productmajorversion}}-testing'`, `'{{site.productmajorversion}}-dev'`, `'{{site.productminorversion}}'`
```
I missed some here sorry.

**Pull request merged**